### PR TITLE
Search bar font

### PR DESCRIPTION
**Pull request by enchev**

### CODE REVIEW

File: ui/styling/stylers.ios.ts [hunk 1]
```diff
@@ -437,6 +437,33 @@ export class SearchBarStyler implements definition.stylers.Styler {
         }
     }
 
+    // font
+    private static setFontInternalProperty(view: view.View, newValue: any, nativeValue: any) {
+        var bar = <UISearchBar>view.ios;
+        var sf = <UITextField>bar.valueForKey("_searchField");
```
Comment (fealebenpae) [inline ui/styling/stylers.ios.ts]: I think using private fields of UIKit classes has the risk of breaking in a future OS release.

Comment (enchev) [inline ui/styling/stylers.ios.ts]: Do we have any alternative? I'll gladly replace this!

**Comment by enchev** [comment]
run ci

File: ui/styling/stylers.ios.ts (hunk 1 above)
Comment (fealebenpae) [inline ui/styling/stylers.ios.ts]: Ideally we'd base our CSS on UIAppearance, which is the iOS HIG-compliant way of doing things, but that would mean supporting things only UIAppearance does, or using custom controls. [Pixate](https://github.com/Pixate/pixate-freestyle-ios) actually does this with dynamic subclassing.

In this particular instance we can do something like `[[UITextField appearanceWhenContainedIn:[UISearchBar class], nil] setFont:font]`, but that would change all text fields in search bars across the application, so it wouldn't be as granular as CSS with its ids and classes.

**Comment by fealebenpae** [comment]
:+1:

File: apps/tests/ui/search-bar/search-bar-tests.ts [hunk 2]
```diff
@@ -58,6 +58,22 @@ export var testSearchBarHintColorAndroid = function () {
...
+
+        searchBar.text = "";
+        searchBar.hint = "hint font-size test";
+
+        var expectedValue = 30;
+        var actualValue;
+
+        searchBar.style.fontSize = expectedValue;
+        actualValue = searchBarTestsNative.getNativeFontSize(searchBar);
+        TKUnit.assert(actualValue === expectedValue, "Actual: " + actualValue + "; Expected: " + expectedValue);
```
Comment (hshristov) [inline apps/tests/ui/search-bar/search-bar-tests.ts]: We have TKUnit.assertEqual - method for easier/better error reporting :)

**Comment by hshristov** [comment]
:+1:

**Pull request merged**